README.md

```diff
@@ -53,6 +57,17 @@ docker run -p 1304:1304 nutsfoundation/nuts-demo-ehr:main
 #### Configuration
 When running in Docker without a config file mounted at `/app/server.config.yaml` it will use the default configuration.
 
+### Starting the _Smart On FHIR_ backend
+
+Follow the steps in [this guide](https://github.com/smart-on-fhir/smart-dev-sandbox#start-the-dev-sandbox).
```

Comment:
I'd suggest running just the Docker container it actually needs, rather than the complete sandbox:

```
docker run -p 4004:8080 smartonfhir/hapi-5:r4-synthea
```

We can always refer to the sandbox we took it from.